src/supported-value.js

```diff
@@ -37,9 +38,16 @@ export default function supportedValue(property, value) {
   // We want only prefixable values here.
   if (typeof value !== 'string' || !isNaN(parseInt(value, 10))) return value
 
+  let postfix = ''
+
+  if (importantRegExp.test(value)) {
```

Comment:
Similar to https://github.com/cssinjs/jss/pull/631/files#r150909712

Reply:
I think we can try to use `el.setAttribute('style', 'color: red !important')` to check value as is. it will help to avoid additional match and replace value.

Reply:
need to check what happens in that case

Reply:
there is also style.cssText, but I think it will be slower

Reply:
you right, I got following results in chrome:

```
element.style.property x 1,272,489 ops/sec ±1.16% (62 runs sampled)
element.setAttribute x 588,921 ops/sec ±1.07% (61 runs sampled)
element.style.cssText x 854,475 ops/sec ±1.46% (60 runs sampled)
```

https://github.com/andrepolischuk/element-style-test/blob/master/index.js